.ci/Jenkinsfile

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env groovy
+
+// Verified with Jenkins v2.164.3
+
+// TODO:
+// 1. Calculate taskset/affinity for the scripts based on total number of jenkins executors
+
+pipeline {
+    agent {label "ml-test-node-gpu"}
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr:'10'))
+        timeout(time: 15, unit: 'MINUTES')
+        disableConcurrentBuilds()
+    }
+
+    environment {
+        ARTEFACT_DIR = "${WORKSPACE}/artefacts"
+        NFS_WORKSPACE = "${NFS_WORKSPACE_ROOT}/ml-nccl-rdma-sharp-plugins-pr/${BUILD_NUMBER}"
```

Comment:
what is cleaning it up?

Reply:
Separate [cleanup job](http://hpc-master:8080/job/_ML_DevOps_team/view/Main/job/ml-daily-workspace-cleaner/) - it daily cleans up 5 day old files located in ${NFS_WORKSPACE_ROOT}.

Reply:
maybe add a comment inline or in the readme file

Reply:
Added the inline comment: `Note: ${NFS_WORKSPACE_ROOT} is cleaned up daily (5 day old files are removed)`